src/interaction/InteractionManager.js

```diff
@@ -1287,226 +1195,96 @@ export default class InteractionManager extends EventEmitter
-     * @param {boolean} hit - the result of the hit test on the display object
-     */
-    processTouchMove(displayObject, hit)
-    {
-        if (!this.moveWhenInside || hit)
+        // This is the way InteractionManager processed touch events before the refactoring, so I've kept
+        // it here. But it doesn't make that much sense to me, since mapPositionToPoint already factors
+        // in this.resolution, so this just divides by this.resolution twice for touch events...
+        if (navigator.isCocoonJS && event.pointerType === 'touch')
         {
-            this.dispatchEvent(displayObject, 'touchmove', this.eventData);
+            interactionData.global.x = interactionData.global.x / this.resolution;
+            interactionData.global.y = interactionData.global.y / this.resolution;
         }
-    }
 
-    /**
-     * Grabs an interaction data object from the internal pool
-     *
-     * @private
-     * @param {Touch} touch - The touch data we need to pair with an interactionData object
-     * @return {PIXI.interaction.InteractionData} The built data object.
-     */
-    getTouchData(touch)
-    {
-        const touchData = this.interactiveDataPool.pop() || new InteractionData();
-
-        touchData.identifier = touch.identifier;
-        this.mapPositionToPoint(touchData.global, touch.clientX, touch.clientY);
-
-        if (navigator.isCocoonJS)
+        // Not really sure why this is happening, but it's how a previous version handled things
+        if (pointerEvent.pointerType === 'touch')
         {
-            touchData.global.x = touchData.global.x / this.resolution;
-            touchData.global.y = touchData.global.y / this.resolution;
+            pointerEvent.globalX = interactionData.global.x;
+            pointerEvent.globalY = interactionData.global.y;
         }
 
-        touch.globalX = touchData.global.x;
-        touch.globalY = touchData.global.y;
+        interactionData.originalEvent = pointerEvent;
+        interactionEvent._reset();
 
-        return touchData;
-    }
-
-    /**
-     * Returns an interaction data object to the internal pool
-     *
-     * @private
-     * @param {PIXI.interaction.InteractionData} touchData - The touch data object we want to return to the pool
-     */
-    returnTouchData(touchData)
-    {
-        this.interactiveDataPool.push(touchData);
+        return interactionEvent;
     }
 
     /**
      * Ensures that the original event object contains all data that a regular pointer event would have
      *
      * @private
-     * @param {TouchEvent|MouseEvent} event - The original event data from a touch or mouse event
+     * @param {TouchEvent|MouseEvent|PointerEvent} event - The original event data from a touch or mouse event
      */
     normalizeToPointerData(event)
     {
-        if (this.normalizeTouchEvents && event.changedTouches)
+        const type = typeof event;
```

Comment:
I don't think that typeof works this way - it would probably just return `"object"`.